test/integration/api/ps.bats

```diff
@@ -158,3 +158,27 @@ function teardown() {
 	[[ "${output}" == *"node-1/c2"* ]]
 	[[ "${output}" != *"node-1/c3"* ]]
 }
+
+@test "docker ps --filter network" {
+	run docker --version
+	if [[ "${output}" == "Docker version 1.9"* || "${output}" == "Docker version 1.10"* ]]; then
+		skip
+	fi
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker_swarm network create -d bridge test1
+	docker_swarm network create -d bridge test2
+
+	sleep 5
+
+	docker_swarm run --name c1 -e constraint:node==node-0 --net node-0/test1 -d busybox:latest sleep 100
+	docker_swarm run --name c2 -e constraint:node==node-1 --net node-1/test1 -d busybox:latest sleep 100
```

Comment:
This line is consistently failing in the PR runs with "network node-1/test1 not found":

https://jenkins.dockerproject.org/job/Swarm-PRs%20(engine%201.13)/185/console

Any idea what is going on here? Shouldn't we be able to attach to newly-created networks right away?